src/zeit/workflow/publishinfo.py

```diff
@@ -56,7 +56,10 @@ def can_publish(self):
     def matches_blacklist(self):
         config = zope.app.appsetup.product.getProductConfiguration(
             'zeit.workflow')
-        blacklist = re.split(', *', config['blacklist'])
+        config_entry = config.get('blacklist')
+        if not config_entry:
+            return False
+        blacklist = re.split(', *', config_entry)
         path = urlparse.urlparse(self.context.uniqueId).path
         for item in blacklist:
             if path.startswith(item):
```

Comment:
Kürzer wäre `if item and path.startswith(item)`, und das würde auch sowas wie trailing comma etc. fangen, oder?